components/map.jsx

```diff
@@ -9,60 +9,44 @@ var Map = React.createClass({
   },
   componentDidMount: function() {
     require('mapbox.js'); // this will automatically attach to Window object.
-    this.map = L.mapbox.map(this.getDOMNode(), this.props.mapId, {
-      accessToken: this.props.accessToken
-    }).setView([43.597, -79.6139], 12);
+    require('leaflet.markercluster');
 
-    var MapLayer = L.mapbox.featureLayer(this.mapId, {accessToken: this.props.accessToken}).addTo(this.map);
-    var geoJson = [{
-      // this feature is in the GeoJSON format: see geojson.org
-      // for the full specification
-      type: 'Feature',
-      geometry: {
-        type: 'Point',
-        // coordinates here are in longitude, latitude order because
-        // x, y is the standard for GeoJSON and many formats
-        coordinates: [-79.6139,
-          43.597
-        ]
-      },
-      properties: {
-        title: 'First Point',
-        description: 'Party description?',
-        "icon": {
-          "iconUrl": "/img/map-marker.svg",
-          "iconSize": [33, 33], // size of the icon
-          "iconAnchor": [15, 15] // point of the icon which will correspond to marker's location
-        }
+    L.mapbox.accessToken = this.props.accessToken;
+    // TODO: The ID in the featureLayer should be change to our internal
+    // GeoJSON once we have the server side database setup.
+    var mb = L.mapbox.featureLayer('examples.map-h61e8o8e');
```

Comment:
Ah ok, so long as the server doesn't need to do any fancy clustering logic itself, I am cool with this... if the server *does* need to be modified to cluster stuff itself, then I have concerns about clustering being a P1 for launch!